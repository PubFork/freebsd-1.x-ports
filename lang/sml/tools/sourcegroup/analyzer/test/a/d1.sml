functor F (A:A) = struct
  val c = C.z
  open A
  open B
  val d = D.y
end
