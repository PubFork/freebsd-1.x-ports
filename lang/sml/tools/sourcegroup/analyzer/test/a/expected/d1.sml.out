source sml "a/d1.sml"
  import structure B C D 
  import signature A 
  export functor F ;

